sdk/tests/conformance2/buffers/get-buffer-sub-data-validity.html

```diff
@@ -239,6 +239,18 @@
     checkGetBufferSubData(gl.INVALID_OPERATION, "noData");
     recreateReadbackBuffer();
 
+    // crbug.com/941930
+    {
+        debug("");
+        debug("write-delete-recreate-fence-wait-read");
+        gl.copyBufferSubData(gl.COPY_READ_BUFFER, gl.COPY_WRITE_BUFFER, 0, 0, 8);
+        deleteReadbackBuffer();
+        recreateReadbackBuffer();
+        await fence();
+        checkGetBufferSubData(gl.NO_ERROR, "noData");
+        deleteReadbackBuffer();
+    }
```

Comment:
Any chance we want to add a stress test which does this multiple times? Does this reliably reproduce the failure the customer found?

Reply:
It's reliable for me. Although it could pass erroneously if the client/service ids line up in the right way.

I added one more test that should (I think) catch it a little more reliably (or at least differently).

Reply:
Looks great.